templates/vue/src/components/Lightbox/media/ActivityMedia/TapestryActivity.vue

```diff
@@ -3,6 +3,7 @@
     <div class="icon"><tapestry-icon :icon="type" /></div>
     <div v-if="type === 'text'" class="text">{{ answerData }}</div>
     <audio v-if="type === 'audio'" controls :src="answerData"></audio>
+    <div v-if="type === 'tasks'">{{ answerData }}</div>
```

Comment:
Can't this be the same as the text div? the text css class does not seems to change much